truffle-config.js

```diff
@@ -24,6 +26,7 @@ module.exports = {
             gasPrice: 20000000000,
             gas: 9500000,
             provider: ganache.provider({
+                db: memdown(),
```

Comment:
You should also add this to `.solcover.js`

Reply:
Done.